test/appium/views/base_view.py

```diff
@@ -316,7 +316,7 @@ def click(self):
         action = TouchAction(self.driver)
         action.press(None, 50, 0).move_to(None, 50, 300).perform()
         super(AirplaneModeButton, self).click()
-        action.press(None, 50, 600).move_to(None, 50, 0).perform()
+        action.press(None, 50, 900).move_to(None, 50, 0).perform()
```

Comment:
sometimes due to appeared notification appium click on it instead of the greyed area - changed coordinates to prevent this